src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -116,6 +138,8 @@ else if (o instanceof Slot)
 				case REMOVE_ALL:
 					assert false;
 			}
+			if (durability && itemStack != null && mode != ChangeMode.RESET && mode != ChangeMode.DELETE)
```

Comment:
Is there a reason RESET/DELETE cases are not supported here? IMO it should be supported

```suggestion
			if (durability && itemStack != null)
```

Reply:
They are supported, reset and delete just set the durability to max, which is done below

Reply:
Okay, I understand you now 👍